catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -138,6 +140,8 @@
 
     private final Associated associated;
 
+    private static final int PAGE_SIZE = 250;
```

Comment:
@mweser Yes it is. I was recommended that value because it requires less mocked results in testing. We should still have the page size, but I'll change it to not be a constant